tests/integration/reqs/test_reliable_submission.py

```diff
@@ -0,0 +1,47 @@
+from tests.integration.integration_test_case import IntegrationTestCase
```

Comment:
`test_reliable_submission` is already under `tests/integration/sugar/test_transaction.py`

Reply:
Ah, thanks, I was wondering why I couldn't find that test. I should've asked :)